docs/configuration/configuration.md

```diff
@@ -272,6 +272,8 @@ See below for the configuration options for Azure discovery:
 
 ```yaml
 # The information to access the Azure API.
+# The Azure environment.
+environment: <string>
```

Comment:
This seems to be `[ environment: <string> | default = "AzurePublicCloud" ]` :-)

Reply:
oh yes :(. Thanks for pointing it out.